app/services/rethinkdb/activities.js

```diff
@@ -91,7 +91,7 @@ var Activities = module.exports = {
       }
 
       // Wykonaj zapytanie do bazy danych
-      r.table(tableName).get(id).update(body, {returnChanges: true}).run(conn, function (err, resp) {
+      r.table(tableName).get(id).replace(body, {returnChanges: true}).run(conn, function (err, resp) {
```

Comment:
dzięki za zauważenie, myślę, że to jest błąd. (Spowodowany prawdopodobnie tym, że próbowałem rozwiązać napotkatkany problem w zły sposób, a zapomniałem tego cofnąć). Trzeba poprawić